nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -618,17 +624,18 @@ contract StakingEscrow is Issuer {
         uint256 oldValue = subStake.lockedValue;
         subStake.lockedValue = oldValue.sub(_newValue);
         require(subStake.lockedValue >= minAllowableLockedTokens);
-        saveSubStake(info, subStake.firstPeriod, 0, subStake.periods.add16(_periods), _newValue);
+        uint16 requestedPeriods = subStake.periods.add16(_periods);
```

Comment:
`add16()` --> `+`?

Reply:
same as previous with `_value`, `_period` could be too big